demos/notebooks/demo_pipeline.ipynb

```diff
@@ -387,18 +613,22 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "#%% plot contours of found components\n",
-    "Cn = cm.local_correlations(images.transpose(1,2,0))\n",
-    "Cn[np.isnan(Cn)] = 0\n",
-    "cnm.estimates.plot_contours_nb(img=Cn)"
+    "border_to_0 = 0 if mot_correct.border_nan == 'copy' else mot_correct.border_to_0 # trim border against NaNs\n",
+    "mc_memmapped_fname = cm.save_memmap(mot_correct.mmap_file, \n",
+    "                                        base_name='memmap_', \n",
+    "                                        order='C',\n",
+    "                                        border_to_0=border_to_0,  # exclude borders, if that was done\n",
+    "                                        dview=multiprocessing_pool)\n",
+    "\n",
+    "Yr, dims, num_frames = cm.load_memmap(mc_memmapped_fname)\n",
+    "images = np.reshape(Yr.T, [num_frames] + list(dims), order='F') #reshape frames in standard 3d format (T x X x Y)"
    ]
   },
   {
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "## Re-run (seeded) CNMF  on the full Field of View  \n",
-    "You can re-run the CNMF algorithm seeded on just the selected components from the previous step. Be careful, because components rejected on the previous step will not be recovered here."
+    "Restart the cluster to clean up memory in prep for CNMF run."
```

Comment:
prep -> preparation (nonnatives of English might not recognise this abbreviation)